commcare_export/excel_query.py

```diff
@@ -180,16 +180,18 @@ def compile_source(worksheet):
         return FlatMap(source=api_query,
                        body=Reference(str(data_source_jsonpath)))
 
-def compile_sheet(worksheet, mappings=None, missing_value=None):
+def compile_sheet(worksheet, mappings=None, missing_value=None, ignore_empty=True):
     mappings = mappings or {}
     source_expr = compile_source(worksheet)
 
     output_table_name = worksheet.title
-    output_headings = get_column_by_name(worksheet, 'field') # It is unfortunate that this is duplicated here and in `compile_fields`
+    output_headings = get_column_by_name(worksheet, 'field')  # It is unfortunate that this is duplicated here and in `compile_fields`
```

Comment:
This comment might be outdated- I don't see a similar line in `compile_fields`

Reply:
The comment is referring to the same call on line [103](https://github.com/dimagi/commcare-export/pull/85/files#diff-a06a76e2710b6183b752bc4e0cfacb87R103):
```
fields = without_empty_tail(get_column_by_name(worksheet, 'field') or [])
```

Probably not a very useful comment though. I'll remove it.

Reply:
2b1854d